lib/plausible/site/schema.ex

```diff
@@ -101,6 +101,39 @@ defmodule Plausible.Site do
     change(site, imported_data: nil)
   end
 
+  @doc """
+  Returns the date of the first recorded stat in the timezone configured by the user.
+  This function does 2 transformations:
+    UTC %NaiveDateTime{} -> Local %DateTime{} -> Local %Date
+
+  ## Examples
+
+    iex> Plausible.Site.local_start_date(%Plausible.Site{stats_start_date: nil})
+    nil
+
+    iex> utc_start = ~N[2022-09-28 00:00:00]
+    iex> tz = "Europe/Helsinki"
+    iex> site = %Plausible.Site{stats_start_date: utc_start, timezone: tz}
+    iex> Plausible.Site.local_start_date(site)
```

Comment:
You can use `...>` to tell this is the same IEx session from the first line:

```suggestion
    ...> tz = "Europe/Helsinki"
    ...> site = %Plausible.Site{stats_start_date: utc_start, timezone: tz}
    ...> Plausible.Site.local_start_date(site)
```